composer.json

```diff
@@ -9,12 +9,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.0.*|5.1.*"
+        "illuminate/support": "4.2.*|5.0.*|5.1.*|5.2.*"
```

Comment:
And? What if 5.3 is backwards incompatible with something. You're assuming it won't break (it very likely will).